veracruz-utils/src/der.rs

```diff
@@ -0,0 +1,151 @@
+//! Some simple manipulations of Distinguished Encoding Rules (DER).
+//!
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Licensing and copyright notice
+//!
+//! See the `LICENSE_MIT.markdown` file in the Veracruz root directory
+//! for information on licensing and copyright.
+
+/// Convert a raw signature (64 bytes) into DER encoding:
+/// two INTEGERs in a SEQUENCE.
+pub fn sig_to_der(sig: Vec<u8>) -> Result<Vec<u8>, ()> {
+    if sig.len() != 64 {
+        Err(())
+    } else {
+        let x = &sig[0..32];
+        let y = &sig[32..64];
+        let xlen = 32 + x[0] / 128;
```

Comment:
Where do all of these constants come from?  Can they be hoisted out to the top-level as defined constants so that they're self-documenting?